CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/GenericJsonIO/JsonIO.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.functionality.GenericJsonIO;
```

Comment:
Package names should be lowercase. Could I also suggest moving this out of the Core Functionality module - this looks generic enough to be moved into Core Utilities.

Reply:
Cool I'll try moving it into core utilities